### PR TITLE
Fix a crash for add to homescreen toolbar button

### PR DESCRIPTION
**Pull request by jdragonbae**
This patch fixes a crash when clicking add to homescreen toolbar button.
According to upstream code changes, call onOptionsItemSelected with the
bundle param which has string resource for title.

Refs: #879

### CODE REVIEW

**Comment by jdragonbae** [comment]
@romandev, @codeimpl I'm not sure whether the way is right. PTAL.

File: banana_cake/java/org/banana/cake/CakeToolbarManager.java [hunk 1]
```diff
@@ -117,7 +120,10 @@ public void findInPage() {
     public void addToHomeScreen() {
         ChromeTabbedActivity activity = getChromeTabbedActivity();
         if (activity == null) return;
-        activity.onMenuOrKeyboardAction(R.id.add_to_homescreen_id, false);
+        Bundle bundle = new Bundle();
+        bundle.putInt(
+                AppBannerManager.MENU_TITLE_KEY, AppMenuVerbiage.APP_MENU_OPTION_ADD_TO_HOMESCREEN);
```
Comment (codeimpl) [inline banana_cake/java/org/banana/cake/CakeToolbarManager.java:125]: https://source.chromium.org/chromium/chromium/src/+/master:chrome/android/java/src/org/chromium/chrome/browser/app/appmenu/AppMenuPropertiesDelegateImpl.java;drc=57407e785f8c1d68da133c609c9adafe55444adb;bpv=1;bpt=1;l=790

**Comment by romandev** [comment]
@jdragonbae Could you please update this?

**Comment by jdragonbae** [comment]
@romandev @codeimpl Sorry for the late. PTAL

**Review by codeimpl — APPROVED** [review]
LGTM

**Review by romandev — APPROVED** [review]
LGTM

**Pull request merged**